woocommerce/google-listings-and-ads
# Fix syntax errors in JSDoc and the incorrect relative paths when generating the tracking README

### Changes proposed in this Pull Request:

- Fix syntax errors in the JSDoc of `TreeSelectControl`.
- Fix the incorrect relative paths when generating the tracking README.
- Prevent the false warnings `@ignore tag does not permit a value` from showing when running `doc:tracking` script.
- Update the tracking README file.

### Detailed test instructions:

1. Run `npm run doc:tracking` to generate tracking README. The script should be able to run without errors.
2. Go to the [tracking README](https://github.com/woocommerce/google-listings-and-ads/blob/fb034f12fbc1411fb8568362a54794d66076f4c2/src/Tracking/README.md) to check if the hyperlinks point to the right source file.

### Changelog entry
